src/components/versionSelector/style.module.scss

```diff
@@ -0,0 +1,128 @@
+.select {
+  display: flex;
+  padding: 0.5rem;
```

Comment:
L: I'm not sure how we proceed with spaces and such based on our design guidelines in the CSS files. I saw that we have `space()` for that in CSS-in-JS.

Reply:
Yeah I agree that we should use a spacing system in docs. Since we have partly moved to tailwind in docs I would rather switch all the spacing definitions there. Or get rid of tailwind. But just streamline all styles at some point :D